package.json

```diff
@@ -45,16 +46,17 @@
   },
   "devDependencies": {
     "hapi": "^17.8.4",
-    "nyc": "^13.1.0",
-    "sinon": "^7.2.2",
-    "standard": "^12.0.1",
-    "tap-spec": "^5.0.0",
-    "tape": "^4.9.1"
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "nyc": "^14.1.1",
+    "sinon": "^7.3.2",
+    "standard": "^13.0.2",
+    "mocha": "^6.1.4"
   },
   "dependencies": {
     "crumb": "^7.2.3",
     "determination": "^2.0.0",
-    "hapi-pino": "^5.2.0",
+    "hapi-pino": "^6.0.1",
```

Comment:
Was this intentional? Seems like it's supposed to be part of https://github.com/homeaway/catalyst-server/pull/1.

Reply:
Thanks - it was intentional but your comment made me realize that this will be safer to do in #1 since that is a breaking change anyway :)